app/extensions/elasticsearch/__init__.py

```diff
@@ -228,9 +228,8 @@ def elasticsearch(cls, search, app=None, total=False, *args, **kwargs):
 
     @classmethod
     def bulk_class(cls):
-        from app.extensions.git_store import GitStore
-
-        bulk_cls = GitStore if issubclass(cls, GitStore) else cls
+        mapper_args = getattr(cls, '__mapper_args__', {})
```

Comment:
Lol, I had no chance of finding that little gotcha :rofl:

Reply:
same!